bundle/manifests/kuadrant.kuadrant.io_authpolicies.yaml

```diff
@@ -2,21 +2,21 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.8.0
```

Comment:
downgrading??? CAn we keep v0.8.0? OI even better, we could upgrade to v0.9.2 ?? maybe v0.10.0?